### PR TITLE
WC-977 Implemented support of foreign key to account ID.

### PR DESCRIPTION
**Pull request by akuznetsov-gridgain**

### CODE REVIEW

File: modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/repositories/NotebooksRepository.java [hunk 1]
```diff
@@ -21,58 +21,71 @@
...
     public NotebooksRepository(Ignite ignite, TransactionManager txMgr) {
-        super(ignite, txMgr);
+        this.txMgr = txMgr;
 
         notebooksTbl = new Table<Notebook>(ignite, "wc_notebooks")
             .addUniqueIndex(Notebook::getName, (notebook) -> "Notebook '" + notebook.getName() + "' already exits");
 
         notebooksIdx = new OneToManyIndex(ignite, "wc_account_notebooks_idx");
+
+        accFkIdx = new ForeignKeyIndex(ignite, "wc_accounts_fk_idx");
```
Comment (nva) [inline modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/repositories/NotebooksRepository.java]: What is the difference between the ForeignKeyIndex and the OneToManyIndex? It looks the same

File: modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/common/Utils.java [hunk 2]
```diff
@@ -129,4 +130,12 @@ public static String currentRequestOrigin() {
             .build()
             .toString();
     }
+
+    /**
+     * @param rows Number of rows.
+     * @return Object with number of affected rows.
+     */
+    public static RowsAffected rowsAffected(int rows) {
```
Comment (nva) [inline modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/common/Utils.java]: Why we need this. At frontend code we don't use it.

**Review by nva — CHANGES_REQUESTED** [review]

File: modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/common/Utils.java (hunk 2 above)
Comment (akuznetsov-gridgain) [reply modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/common/Utils.java]: Removed RowsAffected  class.

File: modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/repositories/NotebooksRepository.java (hunk 1 above)
Comment (akuznetsov-gridgain) [reply modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/repositories/NotebooksRepository.java]: No, they are different.  OneToManyIndex holds <UUID to set of UUIDs>. And ForeignKeyIndex holds <UUID to UUID> links. Also ForeignKeyIndex  validate add/get/remove operations.

**Review by nva — APPROVED** [review]
Looks good

**Pull request closed without merging**